src/main/java/com/app/backend/repository/DecedentRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.app.backend.repository;
+
+import com.app.backend.model.Decedent;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface DecedentRepository extends JpaRepository<Decedent, Integer> {
```

Comment:
Do we need to use the JpaRepository? JpaRepository is an extension of the CRUDRepository but it’s used when we want to perform operations on a batch.